.github/workflows/release.yml

```
@@ -87,15 +96,33 @@ jobs:
           REPO="${{ github.repository }}"
           echo "repo=${REPO@L}" >> "$GITHUB_OUTPUT"
 
-      - name: Build and push Docker image
+      - name: Generate short SHA
+        id: short_sha
+        run: echo "SHORT_SHA=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:51: Double quote to prevent globbing and word splitting

(shellcheck)